prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/GetAnalysisTask.java

```diff
@@ -105,7 +107,9 @@ else if (result instanceof Result.Success<TaskGroup> success) {
 						"[Migration analysis report](%s) completed.".formatted(reportURL));
 				return new DefaultWorkReport(WorkStatus.COMPLETED, workContext);
 			}
-			else if ("Failed".equals(success.value().state())) {
+			else if ("Failed".equals(success.value().state())
+					|| Arrays.stream(Objects.requireNonNull(success.value().tasks()))
```

Comment:
isn't a failed task sets the top-level taskgroup as failed?  what is the state of the taskgroup in that case? (weird API if so)

Reply:
I tried to create application with the same name of existing one. the top level is succeed but task is failed

Reply:
ack